Week3/homework/help.md

```diff
@@ -0,0 +1,65 @@
+Database Homework Week3 Readme
```

Comment:
Lovely documentation. You can also use markdown language, more info here: https://en.support.wordpress.com/markdown-quick-reference/